src/v1/controllers/EnrollChoicesController.js

```diff
@@ -47,7 +47,9 @@ export default FormController.extend({
         return loc('enroll.choices.optional', 'login');
       case Enums.HAS_REQUIRED_SOME_REQUIRED_ENROLLED: {
         const remainingDays = this.options.appState.get('gracePeriodRemainingDays');
-        return Number.isInteger(remainingDays) && remainingDays >= 0
+
+        // remainingDays is null if there is no grace period, 0 if we are still in grace period
+        return (remainingDays !== null && remainingDays >= 0)
```

Comment:
See https://github.com/okta/okta-signin-widget/pull/770/files#r308359100 -- we were using `isInteger` as a proxy for checking if `remainingDays !== null`. We can assume that if it's not `null` then it will be a number and then assume that its an integer.